package.json

```diff
@@ -57,7 +68,17 @@
     "prettier": "^3.3.3",
     "stylelint": "^16.10.0",
     "stylelint-config-standard": "^36.0.1",
-    "webpack-dev-server": "^5.1.0"
+    "webpack-dev-server": "^5.1.0",
+    "@babel/core": "^7.10.3",
```

Comment:
I think we don't need `@babel/core` specified as a dependency. Things seem to work for me without it anyway.